amp/parse.go

```diff
@@ -76,6 +143,14 @@ func parseInt(value string) int64 {
 	return 0
 }
 
+func parseBoolPtr(value string) *bool {
+	rv, err := strconv.ParseBool(value)
+	if err != nil {
+		return nil
```

Comment:
We may want to reconsider this error handling. If the value is empty a return of nil makes sense, but if the value is not a boolean then perhaps a fatal error message is more appropriate.

Reply:
Done. Modified both `parseBoolPtr` and `parseIntPtr`